lms/djangoapps/grades/models.py

```diff
@@ -2,7 +2,9 @@
 Models used for robust grading.
 
 Robust grading allows student scores to be saved per-subsection independent
-of any changes that may occur to the course after the score is achieved.
+of any changes that may occur to the course after the score is achieved, and
```

Comment:
Nit: For better readability, can you add your additional phrase as a separate sentence?  Maybe something like:

_We also persist course-level grades and automatically update them whenever a student's score changes or the course grading policy changes.  Since they are persisted, course grades are also immune to course content changes._